packages/front-end/pages/settings/index.tsx

```diff
@@ -181,418 +203,533 @@ const GeneralSettingsPage = (): React.ReactElement => {
+                    <strong>Owner:</strong> {organization.ownerEmail}
+                  </div>
                 </div>
               </div>
             </div>
-          </div>
-        </div>
-        <div className="my-3 bg-white p-3 border">
-          <div className="row">
-            <div className="col-sm-3">
-              <h4>North Star Metrics</h4>
-            </div>
-            <div className="col-sm-9">
-              <p>
-                North stars are metrics your team is focused on improving. These
-                metrics are shown on the home page with the experiments that
-                have the metric as a goal.
-              </p>
-              <div className={"form-group"}>
-                <div className="my-3">
-                  <div className="form-group">
-                    <label>Metric(s)</label>
-                    <MetricsSelector
-                      selected={form.watch("northStar.metricIds")}
-                      onChange={(metrics) =>
-                        form.setValue("northStar.metricIds", metrics)
-                      }
-                    />
+            <div className="divider border-bottom mb-3 mt-3" />
+            <div className="row">
+              <div className="col-sm-3">
+                <h4>License</h4>
+              </div>
+              <div className="col-sm-9">
+                <div className="form-group row mb-2">
+                  <div className="col-sm-12">
+                    <strong>Plan type: </strong> {licensePlanText}{" "}
                   </div>
-                  <Field label="Title" {...form.register("northStar.title")} />
                 </div>
+                {showUpgradeButton && (
+                  <div className="form-group row mb-1">
+                    <div className="col-sm-12">
+                      <button
+                        className="btn btn-premium font-weight-normal"
+                        onClick={() => setUpgradeModal(true)}
+                      >
+                        {accountPlan === "oss" ? (
+                          <>
+                            Try Enterprise <GBPremiumBadge />
+                          </>
+                        ) : (
+                          <>
+                            Upgrade to Pro <GBPremiumBadge />
+                          </>
+                        )}
+                      </button>
+                    </div>
+                  </div>
+                )}
+                {!isCloud() && permissions.manageBilling && (
+                  <div className="form-group row mt-3 mb-0">
+                    <div className="col-sm-4">
+                      <div>
+                        <strong>License Key: </strong>
+                      </div>
+                      <div
+                        className={clsx(
+                          "d-inline-block mt-1 mb-2 text-center text-muted",
+                          license ? "" : "text-muted"
```

Comment:
Looks like you have `text-muted` twice here

Reply:
oops!